### DISCUSSION TITLE
Welcome to the Grohe Sense integration for HomeAssistant Discussions!

### DISCUSSION THREAD

**Discussion by Flo-Schilli** [Flo-Schilli/homeassistant-grohe_sense#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by daxyorg** [comment]
Hi,

Let we share our firmware versions.
After email to Grohe' support, they sent me an update to Grohe Guard.
So, my firm versions are:

Grohe Guard: ~~`01.44`~~ ; `02.06` 
Grohe Sense: `01.20`
Blue Home: `01.04`

**Reply by Flo-Schilli** [reply]
Hey @daxyorg

great to hear that you got a new version from Grohe. I'm curious how that was working? Did they assigned it to your account/device and you then finally saw it in the app?

I just created a different discussion for the versions, would be great if you could move/add the firmware information there :)

**Reply by daxyorg** [reply]
Hi, @Flo-Schilli !

I wrote an email to Grohe support and they sent me OTA update to version 02.05.
They assigned the update to my email account / device, as you suggested ;)
